src/wp-includes/class-wp-theme-json.php

```diff
@@ -1792,6 +1792,39 @@ public static function scope_selector( $scope, $selector ) {
 		return $result;
 	}
 
+	/**
+	 * Scopes the selectors for a given style node. This includes the primary
+	 * selector, i.e. `$node['selector']`, as well as any custom selectors for
+	 * features and subfeatures, e.g. `$node['selectors']['border']` etc.
+	 *
+	 * @since 6.6.0
+	 *
+	 * @param string $scope Selector to scope to.
+	 * @param array  $node  Style node with selectors to scope.
+	 *
+	 * @return array Node with updated selectors.
+	 */
```

Comment:
Small docblock change.

```suggestion
	/**
	 * Scopes the selectors for a given style node.
	 * 
	 * This includes the primary selector, i.e. `$node['selector']`, as well as any custom
	 * selectors for features and subfeatures, e.g. `$node['selectors']['border']` etc.
	 *
	 * @since 6.6.0
	 *
	 * @param string $scope Selector to scope to.
	 * @param array  $node  Style node with selectors to scope.
	 * @return array Node with updated selectors.
	 */
```

Reply:
Thanks for cleaning that up @audrasjb, I've committed the suggested improvement.